nzbToMedia.py

```diff
@@ -0,0 +1,800 @@
+#!/usr/bin/env python2
```

Comment:
Remove nzbToMedia.py, this has to be installed by the user. You could add this as a submodule to our project, but I don't want to include its source code...